script/DeploySafeRecovery.s.sol

```diff
@@ -2,25 +2,68 @@
 pragma solidity ^0.8.25;
 
 import { Script } from "forge-std/Script.sol";
+import { console } from "forge-std/console.sol";
 import { SafeRecoverySubjectHandler } from "src/handlers/SafeRecoverySubjectHandler.sol";
 import { EmailRecoveryManager } from "src/EmailRecoveryManager.sol";
 import { UniversalEmailRecoveryModule } from "src/modules/UniversalEmailRecoveryModule.sol";
+import { Verifier } from "ether-email-auth/packages/contracts/src/utils/Verifier.sol";
+import { ECDSAOwnedDKIMRegistry } from
+    "ether-email-auth/packages/contracts/src/utils/ECDSAOwnedDKIMRegistry.sol";
+import { EmailAuth } from "ether-email-auth/packages/contracts/src/EmailAuth.sol";
 
-contract DeploySafeRecoveryScript is Script {
+import { Safe7579 } from "safe7579/Safe7579.sol";
+import { Safe7579Launchpad } from "safe7579/Safe7579Launchpad.sol";
+import { IERC7484 } from "safe7579/interfaces/IERC7484.sol";
+
+// 1. `source .env`
+// 2. `forge script --chain sepolia script/DeploySafeRecovery.s.sol:DeploySafeRecovery_Script
+// --rpc-url $BASE_SEPOLIA_RPC_URL --broadcast -vvvv`
+contract DeploySafeRecovery_Script is Script {
     function run() public {
-        address verifier = 0xEdC642bbaD91E21cCE6cd436Fdc6F040FD0fF998;
-        address dkimRegistry = 0xC83256CCf7B94d310e49edA05077899ca036eb78;
-        address emailAuthImpl = 0x1C76Aa365c17B40c7E944DcCdE4dC6e6D2A7b748;
+        bytes32 salt = bytes32(uint256(0));
```

Comment:
Why is the salt fixed here?

Reply:
Yeah maybe that is inconsistent with the other code. Deployed other contracts with factory which uses salts.

The code to deploy the launchpad and safe7579 contract had a salt like this, so I guess I copied it over